osu.Android/OsuGameActivity.cs

```diff
@@ -17,7 +17,7 @@
 
 namespace osu.Android
 {
-    [Activity(Theme = "@android:style/Theme.NoTitleBar", MainLauncher = true, ScreenOrientation = ScreenOrientation.FullUser, SupportsPictureInPicture = false, ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize, HardwareAccelerated = false, LaunchMode = LaunchMode.SingleInstance, Exported = true)]
+    [Activity(ConfigurationChanges = DEFAULT_CONFIG_CHANGES, LaunchMode = DEFAULT_LAUNCH_MODE, MainLauncher = true, ScreenOrientation = ScreenOrientation.FullUser)]
```

Comment:
I don't think `Exported = true` should be dropped? From what I can tell Android 12 and higher [requires the `Exported` attribute to be explicitly specified as `true`](https://developer.android.com/about/versions/12/behavior-changes-12#exported) for activities with intent filters registered.

Reply:
Remember that issue about duplicate exported attributes: https://github.com/xamarin/xamarin-android/issues/6463.
Xamarin will automatically add `exported = true`. Looking at the final `.apk`, I can confirm that the manifest looks as expected. I wanted to keep the ActivityAttribute as short as possible, so that's why the explicit `Exported = true` was removed.

Reply:
Dunno if it's just me but if defaults between android native and xamarin are divergent (like they are in this case - android native expects explicit, and xamarin silently appends it back to not break users) I'd prefer if it stayed explicitly specified. There's no telling if the `true` default on the xamarin side is transitional or not. (In general I find their decision to silently add it back questionable but alas.)

Going to wait for further opinions on this though.